.travis.yml

```diff
@@ -5,14 +5,15 @@ sudo: required
 env:
   global:
     - TESTS=false
-    - PINS="lwt.dev:https://github.com/mirage/lwt.git#tracing"
+    - PINS="lwt.dev:https://github.com/mirage/lwt.git#tracing mirage"
     - UPDATE_GCC_BINUTILS=1
   matrix:
-    - OCAML_VERSION=4.04 WITH_TRACING=1 POST_INSTALL_HOOK="make MODE=unix && make testrun SUDO=sudo && make clean"
-    - OCAML_VERSION=4.04 WITH_TRACING=1 POST_INSTALL_HOOK="make MODE=xen  && make clean"
-    - OCAML_VERSION=4.04 POST_INSTALL_HOOK="make MODE=ukvm   && make clean"
-    - OCAML_VERSION=4.04 POST_INSTALL_HOOK="make MODE=virtio && make clean"
```

Comment:
I'm not sure about removing 4.04 from CI yet... I'd wait until (a) mirage-xen works on 4.06+ and (b) 4.07 is released and working with MirageOS